apcd-cms/src/apps/registrations/templates/submission_form/registration_form_body.html

```diff
@@ -268,14 +269,14 @@ <h5 id="entity_header_{{ forloop.counter }}_{{r.reg_id}}">Entity {{ forloop.coun
                 <div class="field-wrapper textinput">
                     <div class="field-errors" style="display: none"></div>
 
-                    <label for="fein_{{forloop.counter}}_{{r.reg_id}}"> FEIN<sup>5</sup> </label>
+                    <label for="fein_{{ent_no}}_{{r.reg_id}}"> FEIN<sup>5</sup> </label>
```

Comment:
The modal may need access to the JS that allows only one of the Number/Codes to be filled out (FEIN, License, or company code). The modal is requiring all three to be filled out.

Reply:
Should be fixed now!